src/applications/appeals/10182/validations.js

```diff
@@ -1 +1,15 @@
+import { optInErrorMessage } from './content/OptIn';
+
 export const isValidDate = date => date instanceof Date && isFinite(date);
+
+export const requireIssue = (err, fieldData /* , formData */) => {
```

Comment:
can formData just be removed from the parameters here?

Reply:
To do 😉  - thanks!